python/sparknlp/annotator.py

```diff
@@ -2429,6 +2429,9 @@ class ContextSpellCheckerModel(AnnotatorModel):
                   "Controls the influence of individual word frequency in the decision.",
                   typeConverter=TypeConverters.toFloat)
 
+    correctSymbols = Param(Params._dummy(), "correctSymbols", "Whether to correct special symbols or skip spell checking for them", typeConverter=TypeConverters.toBoolean)
+
+    compareLowcase = Param(Params._dummy(), "compareLowcase", "If true will compare tokens in low case with vocabulary", typeConverter=TypeConverters.toBoolean)
```

Comment:
Maybe we can put some name that will make more evident that this is something affecting error detection and speed. 
compareLowerCaseOnDetection() ?
lowerCaseOnDetection?

We know users don't read the API docs... :)